modular_skyrat/code/datums/interactions/lewd/_lewd.dm

```diff
@@ -1647,70 +1647,6 @@
 			var/client/cli = M.client
```

Comment:
Might be best to just delete this entire file, ERP verbs are cringe. /me's are better.